src/components/Gradebook/index.jsx

```diff
@@ -220,13 +231,33 @@ export default class Gradebook extends React.Component {
           return acc;
         }, {});
 
-      const totals = { total: `${this.roundGrade(entry.percent * 100)}/100` };
+      const totals = { Total: `${this.roundGrade(entry.percent * 100)}/100` };
       return Object.assign(results, assignments, totals);
     }),
   };
 
   lmsInstructorDashboardUrl = courseId => `${configuration.LMS_BASE_URL}/courses/${courseId}/instructor`;
 
+  formatHeadings = () => {
+    let headings = [...this.props.headings];
+
+    if (headings.length > 0) {
+      const userInformationHeadingLabel = (
+        <div>
+          <div>Username</div>
+          <div className="font-weight-normal student_key">Student Key</div>
```

Comment:
TODO: The student key heading should not be display if there are no student keys in the data.